plugin.video.sms/resources/lib/client.py

```diff
@@ -0,0 +1,124 @@
+"""
+    Copyright (C) 2020 Scott Ware
+    This file is part of Scoot Media Streamer (plugin.video.sms)
+    SPDX-License-Identifier: GPL-3.0-only
+    See LICENSES/GPL-3.0-only for more information.
```

Comment:
Doesn't exist: use `See license.txt for more information`

Reply:
@scooter1556 will you make this cosmetic change accross your .py files?  I don't want to delay merging this just because of that as it is ready to go in. Although the change doesn't require you to invest much time

Reply:
Apologies @enen92, I missed this in my inbox. I will get this done ASAP.

Reply:
@enen92 , updated.
Many thanks.

Reply:
@enen92, Apologies I missed this in my inbox. Updated.

Thanks.